File: Modul-2/PRAK202/PRAK202-2310817210010-Harry-Pratama-Yunus.c
#include <stdio.h>

int main(){
    float Pertama,Kedua;
    printf("Masukkan Nilai Pertama : ");
    scanf("%f",&Pertama);
    printf("Masukkan Nilai Kedua   : ");
    scanf("%f",&Kedua);
    float hasil= Pertama+Kedua;
    printf("Hasil dari penjumlahan nilai pertama \"%g\" dan nilai kedua \"%.1f\" adalah \"%.2f\"",Pertama,Kedua,hasil);
    return 0;
}

File: Modul-2/PRAK202/PRAK202-2310817210010-Harry-Pratama-Yunus.py
Pertama=float(input("Masukkan Nilai Pertama\t:"))
Kedua=float(input("Masukkan Nilai Kedua\t:"))
hasil=Pertama+Kedua
if Pertama == int(Pertama):
    Pertama=int(Pertama)
    print("Hasil dari penjumlahan nilai pertama \""+str(Pertama)+"\" dan nilai kedua \""+str(Kedua)+"\" adalah \"{:.2f}\"".format(hasil))
else :
    print("Hasil dari penjumlahan nilai pertama \""+str(Pertama)+"\" dan nilai kedua \""+str(Kedua)+"\" adalah \"{:.2f}\"".format(hasil))